front/src/applications/stdcm/types.ts

```diff
@@ -22,16 +22,30 @@ export type StdcmV2SuccessResponse = Omit<
 > & {
   simulation: Extract<SimulationResponse, { status: 'success' }>;
   rollingStock: LightRollingStock;
+  creationDate: Date;
+  speedLimitByTag: string | undefined;
 };
 
 export type SimulationReportSheetProps = {
   stdcmData: StdcmV2SuccessResponse;
-  pathProperties?: ManageTrainSchedulePathProperties;
-  rollingStockData: RollingStockWithLiveries;
-  speedLimitByTag?: string;
   simulationReportSheetNumber: string;
   mapCanvas?: string;
-  creationDate?: Date;
+  operationalPointsList: OperationalPointsList;
+};
+
+export type OperationalPointsList = OperationalPoints[];
+
+type OperationalPoints = {
```

Comment:
Can you rename it something like stdcmResultOperationalPoint ? Because we already have 2 types which have this name :/

Also, it should be singular. It's the array l36 which contains several OPs !

Reply:
fix